app/__init__.py

```diff
@@ -1,10 +1,72 @@
 # app/__init__.py
 """Initializes the app module"""
 
-from flask import Flask
+import os
+
+# third-party imports
+from flask import Flask, Blueprint
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from flask_restful import Api
+from flasgger import Swagger
+
+# local imports
+from config import app_config
+from app.db import db
+
+from app.api.v2.resources.hello import HelloWorld
+from app.api.v2.resources.auth import RegisterUser, LoginUser, ResetPassword, LogoutUser
+from app.api.v2.resources.business import BusinessCollection, BusinessResource, BusinessReviews
 
-# Initialize the app
 app = Flask(__name__)
+app.config.from_object(app_config[os.environ['FLASK_CONFIG']])
```

Comment:
Use `os.environ.get('FLASK_CONFIG', DEFAULT_VALUE)` incase env variable not present.

Reply:
Noted